.github/issue_template

```diff
@@ -1,17 +1,14 @@
-Please make sure that the issue subject starts with `<package-name>: ` so that it's easily identifiable.
+Please make sure that the issue subject starts with `<package-name>: `
 
-This repo here is ONLY for packages maintained in this repo.  For base packages residing in the same repo as the build system and maintained by core devs, please consider opening tickets there for more timely responses
+This repo here is only for packages maintained in this repo.  For base packages residing in the same repo as the build system and maintained by core devs, please consider opening tickets there for more timely responses
 
- - OpenWrt: https://dev.openwrt.org/newticket
- - LEDE: https://bugs.lede-project.org/
+ - OpenWrt base system: https://bugs.openwrt.org
  - Most LuCI packages: https://github.com/openwrt/luci/issues
 
-Thanks for your contribution
-Please remove this text (before ---) and fill the following template
--------------------------------
+# Issue template (remove lines from top till here)
 
-Maintainer: @<github-user> (find it by checking history of the package Makefile)
```

Comment:
I don't like removing this piece of advice. 
Too often the PRs say maintainer=me (or my ID) , although the has quite another maintainer. Newcomers may have no understanding of the maintainer's role, so they just write "me" there.

I feel that it would be go to keep showing the advice about how to find the maintainer.

Reply:
Agree here - otherwise it looks good to go

Reply:
The advice is added backed to both templates now.  Thanks for the input